liquid.py

```diff
@@ -42,6 +42,7 @@ def main():
         commands.initcollection,
         commands.purge,
         commands.getsecret,
+        commands.importfromdocker,
```

Comment:
Let's call it `importfromdockersetup` because technically we're still running docker.